data_structures/stacks/stock_span_problem.py

```diff
@@ -8,44 +8,44 @@
 """
 
 
-def calculation_span(price, s):
+def calculation_span(price: list[float]):
```

Comment:
```suggestion
def calculation_span(price: list[float]) -> list[float]:
```

Reply:
I changes some bit of code to return the ```list[float]```